libbf.c

```diff
@@ -1710,6 +1710,11 @@ static int __bf_div(bf_t *r, const bf_t *a, const bf_t *b, limb_t prec,
         slimb_t d;
 
         na = n + nb;
+
+        if (na >= (SIZE_MAX / sizeof(limb_t)) - 1) {
```

Comment:
This fails on Windows, since IIRC the limb bits are fewer there.

D:/a/quickjs/quickjs/libbf.c:1714:16: error: comparison is always false due to limited range of data type [-Werror=type-limits]

Reply:
Huh, that's rough, will try to play with it more